api/accounts.md

```diff
@@ -0,0 +1,7 @@
+# `/accounts` API
+
+The accounts API deals with authentication and username-related operations.
+
+## `GET /accounts/checkuusername/<username>`
+
+Gets the status of a _username_, not to be confused with a _user_.  Returns a [username object](definitions/username_object.md).  
```

Comment:
I super appreciate this PR! I'm not sure if this really needs to have its own page, though, since it's only used in the one place, as far as I can tell? I'll leave that as your call though! If you do think it should stay as a file, I'd recommend changing its name to "username status object" - since that's more indicative of its contents and the context in which it's used.

Reply:
Whoa this is old!  Yeah, since this is only used once across the API it would probably be best to just put it inline.  I'll get it together 😄

Reply:
All righty - I fixed it with 53136bd.  Should be ready to merge.  Dang, this is an obscene amount of commits for a single file addition 😳